docs-v2/integrations/all/hubspot.mdx

```diff
@@ -3,29 +3,146 @@ title: HubSpot
+<AccordionGroup>  
+  <Accordion title="✅ Authorization">
+    | Tools | Status | 
+    | - | - | 
+    | Pre-built authorization (OAuth 2) | ✅ |
+    | Credentials auto-refresh | ✅ |
+    | Authorization input validation | ✅ |
+    | Pre-built authorization UI | ✅ |
+    | Custom authorization UI | ✅ |
+    | End-user authorization guidelines | ✅ |
+    | Invalid credentials detection | ✅ |
+  </Accordion>
+  <Accordion title="✅ Read & write data">
+    | Tools | Status | 
+    | - | - | 
+    | Pre-built use-cases | ✅ |
+    | API unification | ✅ |
+    | 2-way sync | ✅ |
+    | Webhooks from Nango on data modifications | ✅ |
+    | Real-time webhooks from 3rd-party API | ✅ |
+    | Proxy requests | ✅ |
+  </Accordion>
+  <Accordion title="✅ Observability & data quality">
+    | Tools | Status | 
+    | - | - | 
+    | HTTP request logging | ✅ |
+    | End-to-type type safety | ✅ |
+    | Data runtime validation | ✅ |
+  </Accordion>
+  <Accordion title="✅ Customization">
+    | Tools | Status | 
+    | - | - | 
+    | Create or customize use-cases | ✅ |
+    | Pre-configured pagination | ✅ |
+    | Pre-configured rate-limit handling | ✅ |
+    | Per-customer configurations | ✅ |
+  </Accordion>
+</AccordionGroup>  
 
-## Getting started
+## Pre-built use-cases
 
--   [How to register an Application](https://developers.hubspot.com/docs/api/working-with-oauth#initiating-an-integration-with-oauth-2-0)
--   [OAuth-related docs](https://developers.hubspot.com/docs/api/oauth-quickstart-guide)
--   [List of OAuth scopes](https://developers.hubspot.com/docs/api/working-with-oauth#scopes)
--   [API](https://developers.hubspot.com/docs/api/overview)
+<AccordionGroup>  
+  <Accordion title="Companies">
+  | Endpoint | Description | Readme | 
+  | - | - | - |
+  | `GET /companies` | Fetches a list of companies from Hubspot | [🔗](https://github.com/NangoHQ/integration-templates/blob/main/integrations/hubspot/syncs/companies.ts)|
+  | `POST /companies` | Creates a single company in Hubspot | [🔗](https://github.com/NangoHQ/integration-templates/blob/main/integrations/hubspot/syncs/companies.ts)|
+  | `PATCH /companies` | Update a single company in Hubspot | [🔗](https://github.com/NangoHQ/integration-templates/blob/main/integrations/hubspot/syncs/companies.ts)|
+  | `DEL /companies` | Deletes a company in Hubspot | [🔗](https://github.com/NangoHQ/integration-templates/blob/main/integrations/hubspot/syncs/companies.ts)|
+  </Accordion>
+  <Accordion title="Contacts"></Accordion>
+  <Accordion title="Deals"></Accordion>
+  <Accordion title="Owners"></Accordion>
+  <Accordion title="Pipelines"></Accordion>
+  <Accordion title="Products"></Accordion>
+  <Accordion title="Properties"></Accordion>
+  <Accordion title="Roles"></Accordion>
+  <Accordion title="Tasks"></Accordion>
+  <Accordion title="Others"></Accordion>
+</AccordionGroup>  
+
+<Tip>Not seeing the use-case you need? [Build your own](https://nango.dev/slack) independently.</Tip>
+
+## Pre-requisites
```

Comment:
I never heard of pre-requisites before. I think it would be good to find a more descriptive title

Suggestion:
Requirements & Review Process

Also open to other suggestions!